### PR TITLE
2023-11-17 Gabor Szabo

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

File: _episodes/interviews/2023-11-17-gabor-szabo.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+- [@00:00] - Meet Gabor Szabo - creator of RustDiggar, a website with data collection about Rust crates. Exploring the idea behind RustDiggar and the background of the project
+- [@16:36] - Reasons behind crates without repository links. Discussing the case of the Fastly crate and its missing repository link.
+- [@22:27] - Crates without an owner. Exploring the reasoning associated with crates lacking an owner.
+- [@30:34] - Enhancing RustDiggar and addressing challenges. Exploring potential improvements for RustDiggar, including combating crate name squatting, addressing crate malware, managing crate dependencies, and more.
+- [@38:57] - Outlining the upcoming tasks and developments planned for RustDiggar in the coming months.
```
Comment (jonhoo) [inline _episodes/interviews/2023-11-17-gabor-szabo.md:27]:
```suggestion
- [@00:00] - What is Rust Digger, and why is it useful?
- [@16:36] - Handling crates without repository links (e.g., the Fastly crate)
- [@22:27] - Handling crates without an owner.
- [@30:34] - What's next for Rust Digger, including name squatting, malware, and dependency management.
- [@38:57] - What to expect in the coming months.
```
These all felt overly long and repetitive — how about this shortened version?

**Review by jonhoo — CHANGES_REQUESTED** [review]

File: _episodes/interviews/2023-11-17-gabor-szabo.md (hunk 1 above)
Comment (jonhoo) [reply _episodes/interviews/2023-11-17-gabor-szabo.md]: Committed these, but then marked as unresolved so you'll see them more easily

File: _episodes/interviews/2023-11-17-gabor-szabo.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Rust Digger with Gabor Szabo"
+date: 2023-11-17T07:00:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-11-17-gabor-szabo.md:3]:
```suggestion
date: 2023-11-17T12:26:00Z
```

**Comment by Skgland** [comment]
I still get a 
```
{
  "code": "not_found",
  "message": "File with such name does not exist.",
  "status": 404
}
```
response for the mp3 file, with a `Cache-Control: max-age=31536000` header.
Somehow that was cached (by cloudflare?).
I don't think 404 responses should include such a long `Cache-Control` header.

**Pull request merged**